src/main/java/com/ericsson/ei/subscriptionhandler/SendMail.java

```diff
@@ -107,6 +111,10 @@ public void sendMail(String receiver, String mapNotificationMessage) throws Mess
         return emailAdd;
     }
 
+    public String getSubject(String emailSubject) {
```

Comment:
What do this function do?

Reply:
Well, this function return default subject if one is not provided in the subscription.

Reply:
One liners are cool but it's pretty hard to read, even when you know how it works. I would suggest writing it out instead